src/Simplex/Chat/Types.hs

```diff
@@ -1470,8 +1471,10 @@ type FileTransferId = Int64
 data FileInvitation = FileInvitation
   { fileName :: String,
     fileSize :: Integer,
+    fileDigest :: Maybe FileDigest,
     fileConnReq :: Maybe ConnReqInvitation,
-    fileInline :: Maybe InlineFileMode
+    fileInline :: Maybe InlineFileMode,
+    fileDescrSize :: Maybe Integer -- if this property is set, it means that the sent file will be file description, not the actual file
```

Comment:
how will it be used? if to check size, there's non optional fileSize already; if just as a flag why not use Bool?

Reply:
fileSize includes the size of the file itself, but we need to receive fileDescription first that will arrive in chunks - this is its size, so we know how many chunks and where to truncate.